openmdao/core/driver.py

```diff
@@ -141,6 +141,8 @@ def __init__(self, **kwargs):
 
         self.recording_options.declare('record_metadata', types=bool, default=True,
                                        desc='Record metadata')
```

Comment:
There might be some confusion about what the difference between metadata and model metadata. Wonder if we should be specific on the option name and make it `record_driver_metadata`.  Even if we don't, the desc for it should be more descriptive.

Reply:
Good point. I agree we could do record_driver_metadata but then we would also, to be consistent, have to use record_system_metadata and record_solver_metadata. Do we want to do that?

I did update the docstring to indicate that this option records Driver metadata.